articles/libraries/standard/diagnostics.md

```diff
@@ -121,7 +121,7 @@ Then, using the likelihood function for quantum measurements,
         \right).
 \end{align}
 
-The <xref:microsoft.quantum.diagnostic.assertqubitstate> implements these assertions given representations of $\alpha$ and $\beta$ as values of type <xref:microsoft.quantum.canon.complex>.
+The <microsoft.quantum.diagnostics.assertqubitisinstatewithintolerance> implements these assertions given representations of $\alpha$ and $\beta$ as values of type <xref:microsoft.quantum.math.complex>.
```

Comment:
```suggestion
The <xref:microsoft.quantum.diagnostics.assertqubitisinstatewithintolerance> implements these assertions given representations of $\alpha$ and $\beta$ as values of type <xref:microsoft.quantum.math.complex>.
```
?

Reply:
Good catch, I had missed that the `xref` got dropped; I'll go on and fix that.